### DISCUSSION TITLE
failing to authenticate

### DISCUSSION THREAD

**Discussion by nobody5050** [community/community#9603] [Copilot Conversations]
I've been dealing with this bug since last year and have never successfully used github copilot.
![Screenshot from 2022-01-03 19-46-31](https://user-images.githubusercontent.com/32989720/147998913-520770b9-6a15-4888-8a8e-950e4105d6e8.png)
before you ask, yes I've been accepted.
![Screenshot from 2022-01-03 19-47-48](https://user-images.githubusercontent.com/32989720/147998997-bf00321a-34ec-4473-8850-c6c97cdad42f.png)
This is the output that I get apon trying to activate.
I've tried on many different personal access tokens all with full permissions. every other github extension works fine.

Any ideas to get it working?

**Comment by davecheney** [comment]
Duplicate of https://github.com/github/feedback/discussions/9404
